app/code/Codifi/CustomerRequest/Controller/Request/Save.php

```diff
@@ -0,0 +1,162 @@
+ */
+
+namespace Codifi\CustomerRequest\Controller\Request;
+
+use Magento\Framework\App\Action\Action;
+use Codifi\Training\Model\CustomerNoteFactory;
+use Codifi\Training\Model\NoteRepository;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\Escaper;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Exception\MailException;
+use Magento\Framework\App\Area;
+use Magento\Store\Model\Store;
+use Magento\Framework\Controller\ResultFactory;
+use Codifi\CustomerRequest\Helper\Config;
+
+/**
+ * Class Save
+ * @package Codifi\CustomerRequest\Controller\Request
+ */
+class Save extends Action
+{
+    /**
+     * Customer note factory
+     *
+     * @var CustomerNoteFactory
+     */
+    private $customerNoteFactory;
+
+    /**
+     * Customer note repository
+     *
+     * @var NoteRepository
+     */
+    private $customerNoteRepository;
+
+    /**
+     * Transport builder
+     *
+     * @var TransportBuilder
+     */
+    private $transportBuilder;
+
+    /**
+     * Escaper
+     *
+     * @var Escaper
+     */
+    private $escaper;
+
+    /**
+     * Store manager interface
+     *
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * Save constructor.
+     *
+     * @param Context $context
+     * @param CustomerNoteFactory $customerNoteFactory
+     * @param NoteRepository $customerNoteRepository
+     * @param TransportBuilder $transportBuilder
+     * @param StoreManagerInterface $storeManager
+     * @param Escaper $escaper
+     */
+    public function __construct(
+        Context $context,
+        CustomerNoteFactory $customerNoteFactory,
+        NoteRepository $customerNoteRepository,
+        TransportBuilder $transportBuilder,
+        StoreManagerInterface $storeManager,
+        Escaper $escaper
+    ) {
+        $this->customerNoteFactory = $customerNoteFactory;
+        $this->customerNoteRepository = $customerNoteRepository;
+        $this->transportBuilder = $transportBuilder;
+        $this->storeManager = $storeManager;
+        $this->escaper = $escaper;
+        parent::__construct($context);
+    }
+
+    /**
+     * Execute function
+     *
+     * @return ResultInterface
+     * @throws LocalizedException
+     * @throws MailException
+     * @throws NoSuchEntityException
+     */
+    public function execute(): ResultInterface
+    {
+        $request = $this->getRequest();
+        $customerId = $request->getParam('customer_id');
+        $customerEmail = $request->getParam('email_address');
+        $message = $request->getParam('customer_messsage');
+        $customerName = $request->getParam('customer_name');
+
+        if ($customerId && $customerEmail && $message && $customerName) {
```

Comment:
Actually, it's better to throw exception if one of the required params is missed.